packages/rancher-logging/rancher-logging.patch

```diff
@@ -144,3 +144,7 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-logging/charts-original/values
 +global:
 +  cattle:
 +    systemDefaultRegistry: ""
++
++rke_fluentbit:
++  log_level: "info"
++  mem_buffer_limit: "5MB"
```

Comment:
This could go above under additional logging sources ?

Reply:
that is a good idea